source/extensions/quic_listeners/quiche/active_quic_listener.cc

```diff
@@ -93,10 +95,18 @@ void ActiveQuicListener::onData(Network::UdpRecvData& data) {
 }
 
 void ActiveQuicListener::onReadReady() {
+  if (!enabled()) {
+    ENVOY_LOG(trace, "Quic listener {}: runtime disabled", config_.name());
+    return;
+  }
   quic_dispatcher_->ProcessBufferedChlos(kNumSessionsToCreatePerLoop);
 }
 
 void ActiveQuicListener::onWriteReady(const Network::Socket& /*socket*/) {
+  if (!enabled()) {
+    ENVOY_LOG(trace, "Quic listener {}: runtime disabled", config_.name());
+    return;
+  }
```

Comment:
I think for simplicity we can remove this check? It doesn't matter I think if we get a few last writes out, as long as we don't do any more reads? @danzh2010 wdyt?

Reply:
I would think so too, but either way is fine for me.

Reply:
ok, let me remove the check